app/cms/utils/mdx.tsx

```diff
@@ -318,19 +335,12 @@ function getMdxComponent(page: MdxPage, theme: Theme | null) {
   const { code } = page
 
   const Component = getMDXComponent(code)
-  // const headings = getHeadingsFromMdxComponent(Component);
   function MdxComponent({
     components,
     ...rest
   }: Parameters<typeof Component>["0"]) {
-    return (
-      <div className="container flex flex-row">
-        <div className="mdx-content">
-          {/* @ts-expect-error: We need to figure out how to type this */}
-          <Component components={GetMdxComponents(theme)} {...rest} />
-        </div>
-      </div>
-    )
+    /* @ts-expect-error: TODO: Needs types. */
+    ;<Component components={GetMdxComponents(theme)} {...rest} />
```

Comment:
looks like a `return` was forgotten here?

```suggestion
    return <Component components={GetMdxComponents(theme)} {...rest} />
```